lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -96,10 +96,18 @@ extension CanvasKitExtension on CanvasKit {
   );
 
   // Text decoration enum is embedded in the CanvasKit object itself.
-  external int get NoDecoration;
-  external int get UnderlineDecoration;
-  external int get OverlineDecoration;
-  external int get LineThroughDecoration;
+  @JS('NoDecoration')
+  external double get _NoDecoration;
+  @JS('UnderlineDecoration')
+  external double get _UnderlineDecoration;
+  @JS('OverlineDecoration')
+  external double get _OverlineDecoration;
+  @JS('LineThroughDecoration')
+  external double get _LineThroughDecoration;
+  int get NoDecoration => _NoDecoration.toInt();
+  int get UnderlineDecoration => _UnderlineDecoration.toInt();
+  int get OverlineDecoration => _OverlineDecoration.toInt();
+  int get LineThroughDecoration => _LineThroughDecoration.toInt();
```

Comment:
IMHO, it's better to put each trampoline right under its respective private getter like you did in the rest of the PR:
```suggestion
  @JS('NoDecoration')
  external double get _NoDecoration;
  int get NoDecoration => _NoDecoration.toInt();
  @JS('UnderlineDecoration')
  external double get _UnderlineDecoration;
  int get UnderlineDecoration => _UnderlineDecoration.toInt();
  @JS('OverlineDecoration')
  external double get _OverlineDecoration;
  int get OverlineDecoration => _OverlineDecoration.toInt();
  @JS('LineThroughDecoration')
  external double get _LineThroughDecoration;
  int get LineThroughDecoration => _LineThroughDecoration.toInt();
```